advisor-ui/src/components/OpportunityCard/OpportunityCard.jsx

```diff
@@ -3,10 +3,19 @@ import defaultImg from "../../assets/volunteer-opportunities-ideas-article-1200x
 import "./OpportunityCard.css";
 import OpportunityModal from "../OpportunityModal/OpportunityModal";
 import Modal from "../Modal/Modal";
+import Chip from "@mui/material/Chip";
+import Grow from "@mui/material/Grow";
 
 export default function OpportunityCard({ eventItem }) {
   const [isOpen, setIsOpen] = useState(false);
 
+  let backgroundColor = "#8BC34A";
```

Comment:
Maybe, you should define color as constants somewhere in a file so that they can be reused at other places without code duplication.